packages/chakra-ui-core/src/CAccordion/CAccordion.stories.js

```diff
@@ -2,6 +2,17 @@ import { storiesOf } from '@storybook/vue'
 import { CBox, CAccordion, CAccordionItem, CAccordionHeader, CAccordionPanel, CAccordionIcon } from '..'
 
 storiesOf('UI | Accordion', module)
+  .add('Basic TEst', () => ({
```

Comment:
Basic test?

Reply:
I usually use Storybook in development so I must have forgotten to remove it. Thanks